src/briefcase/commands/create.py

```diff
@@ -20,7 +20,7 @@ class InvalidTemplateRepository(BriefcaseCommandError):
     def __init__(self, template):
         self.template = template
         super().__init__(
-            'Unable to clone application template; is the template path {template!r} correct?'.format(
+            "Unable to clone application template; is the template path {template!r} correct?".format(
```

Comment:
I accidentally-ish ran `black` on this file. I can separate that into a follow-up commit. Is that the right code formatter to use? I like black, but I am happy to defer to whatever you advise.

Reply:
We haven't *formally* adopted black, but we're definitely leaning towards black style choices. At some point (probably in the not too distant future) we'll almost certainly formally adopt black into the toolchain.

Reply:
+1

I'll keep this pull request more minimal.

I'm interested in volunteering to `black`-ify BeeWare projects, but we can discuss that later.